library/ssl_tls13_keys.c

```diff
@@ -0,0 +1,325 @@
+/*
+ *  TLS 1.3 key schedule
+ *
+ *  Copyright (C) 2006-2020, ARM Limited, All Rights Reserved
```

Comment:
Please use the new copyright statement format: https://github.com/ARMmbed/mbedtls/pull/3546

Reply:
:+1: Done